Content.Server/Corvax/StationGoal/StationGoalPrototype.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Corvax.StationGoal
+{
+    [Serializable, Prototype("stationGoal")]
+    public sealed class StationGoalPrototype : IPrototype
+    {
+        [IdDataFieldAttribute] public string ID { get; } = default!;
+
+        [DataField("text")] public string Text { get; set; } = string.Empty;
```

Comment:
```suggestion
        [DataField("text")] public string Text { get; set; } => Loc.GetString(ID);
```
Obviously the FTL names will need to be changed, but there's no need to write a LOC identifier for each one.

Should it just be the ID? `station-goal-StationGoalStorage` seems silly.

Reply:
Quick note- the way I wrote it here would also involve removing the datafield. It could instead be set to the Loc string *by default*, which would allow overwriting it in YAML, even if you shouldn't.